src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -43,6 +45,7 @@ const AppRouter = () => {
           path="/SparkleVerse"
           component={() => <Redirect to={SPARKLEVERSE_HOMEPAGE_URL} />}
         />
+        <Route path="/loadingTest" component={LoadingPage} />
```

Comment:
[TODO] Should probably remove this test route.. it won't hurt the app, but it sort of makes the code look slightly messy. Maybe we can move it to `/debug/loading` or similar and keep it there?

Reply:
fixed in [`327b240` (#1240)](https://github.com/sparkletown/sparkle/pull/1240/commits/327b240f79d95ae502556f51f4c7dd4920cf3dd4)

Reply:
fixed by renaming route to `/debug/loading` in [`6d73a00` (#1240)](https://github.com/sparkletown/sparkle/pull/1240/commits/6d73a00b7c7282d4c79f7987be5d3dd9ec45ebb7)